packages/stg-evm-v2/devtools/config/mainnet/utils.ts

```diff
@@ -37,6 +37,7 @@ export const onPeaq = withEid(EndpointId.PEAQ_V2_MAINNET)
 export const onPlume = withEid(EndpointId.PLUME_V2_MAINNET)
 export const onPolygon = withEid(EndpointId.POLYGON_V2_MAINNET)
 export const onRarible = withEid(EndpointId.RARIBLE_V2_MAINNET)
+export const onRsk = withEid(EndpointId.ROOTSTOCK_V2_MAINNET)
```

Comment:
```suggestion
export const onRootstock = withEid(EndpointId.ROOTSTOCK_V2_MAINNET)
```

Seems we use `onRootstock` in the config files